src/app/modules/main/chat_section/controller.nim

```diff
@@ -427,9 +427,24 @@ proc getAllChats*(self: Controller, communityId: string): seq[ChatDto] =
   return self.communityService.getAllChats(communityId)
 
 proc getChatsAndBuildUI*(self: Controller) =
-  let channelGroup = self.chatService.getChannelGroupById(self.sectionId)
+  var chats: seq[ChatDto]
+  var community: CommunityDto
+  if self.isCommunity():
+    community = self.getMyCommunity()
+    # chats = community.chats
+    chats = self.chatService.getChatsForCommunity(community.id)
+    echo "name ", community.name
+    echo "CHATS LEN ", chats.len
+    # Fetch community chat members as we lazy load them
+    # TODO add fetch
```

Comment:
Should be resolved before merge?

Reply:
I the end I didn't need to do that. I forgot that we have the members in the community chats already. I just removed the TODO.